### PR TITLE
fix: hide reactions avatars in channels

### LINKED ISSUE #214: Flickering avatar when adding first reaction to a channel post
Reactions in channels are anonymous, however, when you add a first reaction, your avatar will be displayed for a second and then disappear.

### PR DESCRIPTION
**Pull request by iib0011**
fixes #214

### CODE REVIEW

File: src/components/middle/message/Reactions.tsx [hunk 1]
```diff
@@ -49,6 +51,7 @@ const Reactions: FC<OwnProps> = ({
           withRecentReactors={totalCount <= MAX_RECENT_AVATARS}
           genericEffects={genericEffects}
           observeIntersection={observeIntersection}
+          hideAvatars={hideAvatars}
```
Comment (Ajaxy) [inline src/components/middle/message/Reactions.tsx:54]: It looks like `withRecentReactions` already does the same.

File: src/components/middle/message/Reactions.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ type OwnProps = {
   metaChildren?: React.ReactNode;
   genericEffects?: ApiStickerSet;
   observeIntersection?: ObserveFn;
+  hideAvatars?: boolean;
```
Comment (Ajaxy) [inline src/components/middle/message/Reactions.tsx:23]: `noRecentReactors` would be a more convenient name.

**Review by Ajaxy — CHANGES_REQUESTED** [review]

File: src/components/middle/message/Reactions.tsx (hunk 1 above)
Comment (iib0011) [reply src/components/middle/message/Reactions.tsx]: done

**Review by Ajaxy — APPROVED** [review]

**Comment by Ajaxy** [comment]
Thanks, merged!

**Pull request merged**